src/anniversary/sf33rd/AcrSDK/ps2/flps2vram.c

```diff
@@ -1977,9 +2130,41 @@ s32 flPS2DeleteVramList(FLTexture *lpflTexture) {
     return 1;
 }
 
-INCLUDE_ASM("asm/anniversary/nonmatchings/sf33rd/AcrSDK/ps2/flps2vram", flPS2PurgeTextureFromVRAM);
+void flPS2PurgeTextureFromVRAM(u32 th) {
+    FLTexture *lpflTexture;
+
+    lpflTexture = &flTexture[th - 1];
+
+    if (th > 0x100) {
```

Comment:
0x100 should be a macro constant
```suggestion
    if (th > FL_TEXTURE_MAX) {
```